encodings/json/README.md

```diff
@@ -56,14 +56,14 @@ supports easier future extensibility.
 
 The frame should itself be a JSON object, and contain some properties:
 
-| Property        | Requirement | Notes                                                       |
-| --------------- | ----------- | ----------------------------------------------------------- |
-| `events`        | required    | These are the actual events and the whole point of bundling |
-| `start_time`    | optional    | The earliest timestamp_t of any bundled events              |
-| `end_time`      | optional    | The latest timestamp_t of any bundled events                |
-| `start_time_dt` | optional    | The earliest datetime_t of any bundled events               |
-| `end_time_dt`   | optional    | The latest datetime_t of any bundled events                 |
-| `count`         | optional    | The number of bundled events                                |
+| Property        | Data Type      | Requirement | Notes                                                       |
+| --------------- | -------------- | ----------- | ----------------------------------------------------------- |
+| `events`        | array of event | required    | These are the actual events and the whole point of bundling |
```

Comment:
```suggestion
| `events`        | array of objects | required    | These are the actual events and the whole point of bundling |
```
Since this is describing a JSON data structure, perhaps "object" is better  than "event"? 

(I also think the array's item type should be pluralized, but that's a stylistic thing.)

Reply:
Although at the JSON level it is indeed an array of objects, I was trying to clarify that it is not just any object, it is specifically an event in the OCSF sense.  I wound up with the singular version because I was trying to write the identifier used for OCSF events, but it turns out there isn't one that I could find so I wrote event instead of `event`.  There is a `base_event` but that isn't what it is either.

At the end, given that it isn't a literal identifier, it should be plural.

How about "array of OCSF events", or if I can stand to be more verbose, "array of JSON encoded OCSF events"?

Reply:
> "array of OCSF events"

That's clear enough for me!